lib/models/cosmos_wallet.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter_app/models/wallet_type.dart';
+import 'package:sacco/wallet.dart';
+import 'package:flutter_app/models/emeris_wallet.dart';
+
+class CosmosWallet implements EmerisWallet {
+  final Wallet wallet;
+  final WalletDetails baseWalletDetails;
+
+  CosmosWallet({
+    required this.baseWalletDetails,
+    required this.wallet,
+  });
+
+  @override
+  WalletDetails get walletDetails => baseWalletDetails;
+
+  @override
+  WalletType get walletType => WalletType.Cosmos;
+}
```

Comment:
you can simplify it a bit:
```suggestion
class CosmosWallet implements EmerisWallet {
  final Wallet wallet;
  @override
  final WalletDetails walletDetails;

  CosmosWallet({
    required this.walletDetails,
    required this.wallet,
  });

  @override
  WalletType get walletType => WalletType.Cosmos;
}
```

Reply:
Done for both Ethereum and Cosmos wallets ✅